src/app/components/status-button-menu/status-button-menu.component.ts

```diff
@@ -40,4 +89,19 @@ export class StatusButtonMenuComponent implements OnInit {
 
     this.downVoted = !this.downVoted;
   }
+
+  evaluateScore(score,q1,q2,q3,q4){
```

Comment:
Det der bør nok håndteres på backenden. Så skal man kun algoritmen for at bestemme om det er godt eller skidt et enkelt sted (backenden).

Generelt er det bedst at undgå at databehandle på frontenden